articles/openshift/oshift-how-restrict-access-to-openshift-routes-by-ip.md

```diff
@@ -0,0 +1,57 @@
+---
+title: UKCloud Knowledge Centre article about whitelisting public IP's on your openshift routes | UKCloud Ltd
+description: Restrict access to openshift routes by public IP[s]
+services: openshift
+author: Mudasar Hussain
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: restrict access to openshift routes by public IP[s]
+toc_fullpath: How To/restricting-access-to-openshift-routes-by-ip.md
+toc_mdlink: oshift-how-restrict-access-to-openshift-routes-by-ip.md
+---
+
+# How to restrict access to a route in openshift by IP[s] address
+
+## Overview
+
+This knowledge centre article explains how openshift customers can restrict access to the routes they create to their deployed services by IP[s] addresses.
+
+### Intended audience
+
+Openshift developers who are looking to secure exposed routes of application by IP[s]
```

Comment:
OpenShift developers who are looking to secure exposed routes to applications by IP address.

(Don't forget the full stop at the end of the sentence.